Content.Shared/Materials/SharedMaterialReclaimerSystem.cs

```diff
@@ -37,12 +42,28 @@ public override void Initialize()
         SubscribeLocalEvent<MaterialReclaimerComponent, EntityUnpausedEvent>(OnUnpaused);
         SubscribeLocalEvent<MaterialReclaimerComponent, ExaminedEvent>(OnExamined);
         SubscribeLocalEvent<MaterialReclaimerComponent, GotEmaggedEvent>(OnEmagged);
+        SubscribeLocalEvent<MaterialReclaimerComponent, AttemptDamageContactEvent>(OnAttemptDamageContact);
         SubscribeLocalEvent<MaterialReclaimerComponent, MapInitEvent>(OnMapInit);
         SubscribeLocalEvent<CollideMaterialReclaimerComponent, StartCollideEvent>(OnCollide);
         SubscribeLocalEvent<ActiveMaterialReclaimerComponent, ComponentStartup>(OnActiveStartup);
         SubscribeLocalEvent<ActiveMaterialReclaimerComponent, EntityUnpausedEvent>(OnActiveUnpaused);
+        //SubscribeLocalEvent<MobStateChangedEvent>(OnMobStateChanged);
     }
 
+    /*private void OnMobStateChanged(MobStateChangedEvent ev)
+    {
+        if (ev.NewMobState != MobState.Dead)
+            return;
+        TryComp<MaterialReclaimerComponent>(ev.Origin, out var reclaimer);
+        foreach (var entity in _entityLookupSystem.GetEntitiesInRange(ev.Target, 0.4f, LookupFlags.Approximate | LookupFlags.Static))
+        {
+            if (HasComp<MaterialReclaimerComponent>(entity))
+            {
+                TryStartProcessItem(entity, ev.Target, reclaimer);
+            }
+        }
+    }*/
+
```

Comment:
This should be removed entirely rather than being commented out.

Reply:
This is just a temporary thing that I was using when trying to fix a bug/reference. Not gonna stay.